### PR TITLE
Create from dom object

### PR DESCRIPTION
**Pull request by piotr-gawron**
This PR provides simple functionality that allows to create DualListbox from pure DOM element.

It's usefull when the page is created dynamically via JS and some elements might be attached to the document later (or at completely different part of the code).

As an alternative to check if argument to the constructor is a dom element I considered checking if argument is a string. 
Here are some links for reference:
* check if object is a DOM element: https://stackoverflow.com/questions/384286/javascript-isdom-how-do-you-check-if-a-javascript-object-is-a-dom-object
* check if object is string (for selectors): https://stackoverflow.com/questions/4059147/check-if-a-variable-is-a-string

### CODE REVIEW

File: src/dual-listbox.js [hunk 1]
```diff
@@ -19,7 +19,11 @@ const SELECTED_MODIFIER = 'dual-listbox__item--selected';
...
     constructor(selector, options={}) {
         this.setDefaults();
-        this.select = document.querySelector(selector);
+        if (this.isDomElement(selector)) {
+            this.select = selector;
+        } else {
+            this.select = document.querySelector(selector);
+        }
 
         this.selected = [];
```
Comment (JostCrow) [inline src/dual-listbox.js]: It is better to have the variable declarations at the top.

**Comment by JostCrow** [comment]
This is merged in 1.0.6. Thank you for contributing.

@piotr-gawron Can you also add yourself to the contributers?

**Comment by piotr-gawron** [comment]
@JostCrow Thanks, What do you mean by add myself to contributers?

**Comment by svenvandescheur** [comment]
@piotr-gawron If you like, you can add your name to the `CONTRIBUTERS` file in the root of the repo :).

File: src/dual-listbox.js [hunk 2]
```diff
@@ -19,7 +19,11 @@ const SELECTED_MODIFIER = 'dual-listbox__item--selected';
 class DualListbox {
     constructor(selector, options={}) {
         this.setDefaults();
-        this.select = document.querySelector(selector);
+        if (this.isElement(selector)) {
```
Comment (piotr-gawron) [inline src/dual-listbox.js]: Well, it depends how many browsers (and which versions) you want to support:  https://stackoverflow.com/questions/384286/javascript-isdom-how-do-you-check-if-a-javascript-object-is-a-dom-object

Comment (svenvandescheur) [inline src/dual-listbox.js]: We support latest + previous of all major browsers + IE11. Just checked IE11 which seems to support this. I think @JostCrow is working on setting up cross env testing.

Comment (JostCrow) [inline src/dual-listbox.js]: #6 This is a setup of what we want to cover. (we still need to add all the browsers. Also we want to support some mobile browsers.) IE11 will be dropped when it is no longer supported by Microsoft.

**Pull request merged**